test/test-platform-output.c

```diff
@@ -122,5 +123,15 @@ TEST_IMPL(platform_output) {
   }
   uv_free_interface_addresses(interfaces, count);
 
+  err = uv_os_get_passwd(&pwd);
+  ASSERT(err == 0);
+
+  printf("uv_os_get_passwd:\n");
+  printf("  euid: %d\n", pwd.uid);
```

Comment:
Nope. On a second thought, calling the field UID is ok, since we could reuse the structure in the future. Discard the comment.